src/main/java/com/example/codebase/domain/agora/service/AgoraService.java

```diff
@@ -82,10 +82,19 @@ public AgorasResponseDTO getAllAgora(PageRequest pageRequest) {
     }
 
     @Transactional(readOnly = true)
-    public AgoraDetailReponseDTO getAgora(Long agoraId) {
+    public AgoraDetailReponseDTO getAgora(Long agoraId, String username) {
         Agora agora = agoraRepository.findById(agoraId)
                 .orElseThrow(() -> new NotFoundException("존재하지 않는 아고라입니다."));
-        
+
+        Member loginMember = memberRepository.findByUsername(username)
+                .orElseThrow(NotFoundMemberException::new);
+
+        AgoraParticipant loginParticipant = agoraParticipantRepository.findById(AgoraParticipantIds.of(agora, loginMember))
+                .orElse(null);
+
+        if (loginParticipant != null) {
+            return AgoraDetailReponseDTO.of(agora, loginParticipant);
+        }
         return AgoraDetailReponseDTO.from(agora);
```

Comment:
`not equals` 조건보다는 `equals` 더 잘 읽힐것같습니다!
```suggestion
        if (loginParticipant == null) {
            return AgoraDetailReponseDTO.from(agora);
        }
        return AgoraDetailReponseDTO.of(agora, loginParticipant);
```